dsme/dsme-server.c

```diff
@@ -34,13 +34,13 @@
 #define _GNU_SOURCE
 #endif
 
-#include "dsme/mainloop.h"
-#include "dsme/modulebase.h"
-#include "dsme/dsmesock.h"
-#include "dsme/protocol.h"
-#include "dsme/logging.h"
-#include "dsme/messages.h"
-#include "dsme/oom.h"
+#include "../include/dsme/mainloop.h"
+#include "../include/dsme/modulebase.h"
+#include "../include/dsme/dsmesock.h"
+#include <dsme/protocol.h>
```

Comment:
Immediate problem was: I got miffed enough about how difficult it was to find what actually got included -> do mechanical path editing to fix that.

I considered shuffling include order too, but it can cause subtle breakages and must be handled with more care. Given the amount of changed files and little benefit that would be gained by such operation I decided to leave it for another day.